src/Analyzer/TableFunctionNode.cpp

```diff
@@ -36,6 +36,13 @@ void TableFunctionNode::resolve(TableFunctionPtr table_function_value, StoragePt
     unresolved_arguments_indexes = std::move(unresolved_arguments_indexes_);
 }
 
+void TableFunctionNode::resolve(StoragePtr storage_value, ContextPtr context)
+{
```

Comment:
But now `table_function` is not assigned, so method `isResolved` will return `false`, even though `resolve` is called. Seems this will lead to incorrect behaviour? Why cannot we use `resolve` from `void resolve(TableFunctionPtr table_function_value, StoragePtr storage_value, ContextPtr context, std::vector<size_t> unresolved_arguments_indexes_);`?